src/Core/Grid/Query/CustomerGroupsQueryBuilder.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+namespace PrestaShop\PrestaShop\Core\Grid\Query;
+
+use PrestaShop\PrestaShop\Core\Grid\Search\SearchCriteriaInterface;
+
+class CustomerGroupsQueryBuilder extends AbstractDoctrineQueryBuilder
+{
+
+    public function getSearchQueryBuilder(SearchCriteriaInterface $searchCriteria)
+    {
+        $builder = $this->getCustomerGroupsQueryBuilder();
+
+        $builder
+            ->select('g.*')
+        ;
```

Comment:
Usually its better practice to only select what you need in a list, not everything blindly